File: PHP/Efectivo.php
<?php
require_once('pay.php');
class Efectivo extends pay 
{
    public $denominacion;
    public $monedas;
    public $billetes;

    public function __construct($id, $denominacion, $monedas, $billetes)
    {
        parent::__construct($id);

        $this->denominacion = $denominacion;
        $this->monedas = $monedas;
        $this->billetes = $billetes;
    }
    
}

File: Java/Efectivo.java
package Java;

public class Efectivo extends Pay {

    float denominacion;
    float monedas;
    float billetes;
    public Efectivo(Integer id, float denominacion, float monedas, float billetes) {
        super(id);
        this.denominacion = denominacion;
        this.monedas = monedas;
        this.billetes = billetes;
    }

    

}
